webapp/components/Group/GroupForm.vue

```diff
@@ -7,133 +7,184 @@
       :schema="formSchema"
       @submit="submit"
     >
-      <ds-input
-        :label="$t('group.name')"
-        v-model="formData.name"
-        :placeholder="`${$t('group.name')} …`"
-      ></ds-input>
-      <ds-input
-        v-if="update"
-        :label="$t('group.labelSlug')"
-        v-model="formData.slug"
-        icon="at"
-        :placeholder="`${$t('group.labelSlug')} …`"
-      ></ds-input>
-      <!-- groupType -->
-      <!-- TODO: change it has to be implemented later -->
-      <!-- TODO: move 'ds-select' from styleguide to main code and implement missen translation etc. functionality -->
```

Comment:
Would be nice do have this both todos in the code.
Or you don't think so?

Reply:
is now included